charts/gardener/controlplane/values.yaml

```diff
@@ -513,6 +517,9 @@ global:
       enabled: false
       clusterIP: 1.2.3.4
       createNamespace: true
+      admission:
+        user:
+          name: ""
```

Comment:
Maybe you could document the implications of these settings somewhere – just like you did in the extensions :)

Reply:
OK, sounds reasonable. Should I maybe add a document in [this](https://github.com/gardener/gardener/tree/master/docs/deployment) folder or is there any other place that will be more suitable? Have in mind that this changes will also be introduced to gardener-apiserver/scheduler and others(if any).

Reply:
Actually [this](https://github.com/gardener/gardener/blob/master/docs/deployment/setup_gardener.md) looks like a good place to extend. WDYT?

Reply:
> Actually [this](https://github.com/gardener/gardener/blob/master/docs/deployment/setup_gardener.md) looks like a good place to extend. WDYT?

Yes, that looks good. 👍 
Don't worry too much about the place. We can still move it around later on, if we find some better place, e.g. when adding the same for the other control plane components.